alphacsc/learn_d_z_multi.py

```diff
@@ -200,11 +200,10 @@ def compute_d_func(X, z_hat, D_hat, constants):
                             uv_constraint=uv_constraint, loss=loss,
                             loss_params=loss_params, window=window, **d_kwargs)
 
-    with get_z_encoder_for(solver_z, z_kwargs, X, D_hat, n_atoms,
-                           n_times_atom, algorithm, reg, loss,
+    with get_z_encoder_for(X, D_hat, n_atoms, n_times_atom, n_jobs,
+                           solver_z, z_kwargs, algorithm, reg, loss,
                            loss_params, uv_constraint,
```

Comment:
Use keyword argument whenever possible so there is no issue with argument ordering?

Reply:
Maybe to change in another PR :)